generated/microsoftgraph/microsoft.graph/beta/0.1.9-preview/types.md

```diff
@@ -0,0 +1,381 @@
+* **Valid Scope(s)**: Unknown
+### Properties
+* **apiVersion**: 'beta' (ReadOnly, DeployTimeConstant): The resource api version
+* **businessPhones**: string[] (ReadOnly): The telephone numbers for the user. Only one number can be set for this property. Read-only for users synced from on-premises directory
+* **deletedDateTime**: string (ReadOnly): Date and time when this object was deleted. Always null when the object hasn't been deleted.
+* **displayName**: string (ReadOnly): The name displayed in the address book for the user. This value is usually the combination of the user's first name, middle initial, and last name. This property is required when a user is created, and it cannot be cleared during updates. Maximum length is 256 characters
+* **givenName**: string (ReadOnly): The given name (first name) of the user. Maximum length is 64 characters
+* **id**: string (ReadOnly): The unique identifier for an entity. Read-only.
+* **jobTitle**: string (ReadOnly): The user's job title. Maximum length is 128 characters
+* **mail**: string (ReadOnly): The SMTP address for the user, for example, admin@contoso.com. Changes to this property also update the user's proxyAddresses collection to include the value as an SMTP address. This property can't contain accent characters. NOTE: We don't recommend updating this property for Azure AD B2C user profiles. Use the otherMails property instead
+* **mobilePhone**: string (ReadOnly): The primary cellular telephone number for the user. Read-only for users synced from the on-premises directory
+* **officeLocation**: string (ReadOnly): The office location in the user's place of business. Maximum length is 128 characters
+* **preferredLanguage**: string (ReadOnly): The preferred language for the user. The preferred language format is based on RFC 4646. The name combines an ISO 639 two-letter lowercase culture code associated with the language and an ISO 3166 two-letter uppercase subculture code associated with the country or region. Example: 'en-US', or 'es-ES'
+* **surname**: string (ReadOnly): The user's surname (family name or last name). Maximum length is 64 characters
+* **type**: 'Microsoft.Graph/users' (ReadOnly, DeployTimeConstant): The resource type
+* **userPrincipalName**: string (Required, DeployTimeConstant): The user principal name (UPN) of the user. The UPN is an Internet-style sign-in name for the user based on the Internet standard RFC 822. By convention, this should map to the user's email name. The general format is alias@domain, where the domain must be present in the tenant's verified domain collection. This property is required when a user is created. The verified domains for the tenant can be accessed from the verifiedDomains property of organization.NOTE: This property can't contain accent characters. Only the following characters are allowed A - Z, a - z, 0 - 9, '. - _ ! # ^ ~. For the complete list of allowed characters, see username policies
```

Comment:
Not in scope for this PR but there's probably another flag needed for resource identifiers. I will follow up with another PR to make the change